src/mimefactory.rs

```diff
@@ -1002,6 +1002,21 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
     }
 }
 
+/// Returns base64-encoded buffer `buf` split into 78-bytes long
+/// chunks separated by CRLF.
+///
+/// This line length limit is an
+/// [RFC5322 requirement](https://tools.ietf.org/html/rfc5322#section-2.1.1).
+fn wrapped_base64_encode(buf: &[u8]) -> String {
+    let base64 = base64::encode(&buf);
+    let mut chars = base64.chars();
+    (0..)
+        .map(|_| chars.by_ref().take(78).collect::<String>())
+        .take_while(|s| !s.is_empty())
+        .collect::<Vec<_>>()
+        .join("\r\n")
+}
+
```

Comment:
looks good.  not sure but it might be that async-smtp / lettre already wrap things correctly if there is a space where it can insert CRs. Then again i guess it can't hurt to do it ourselves.

Reply:
Our fork of lettre_email uses https://github.com/deltachat/rust-email which does wrapping for headers at least if there are spaces. But I don't see the reason to add spaces when we can add CRLF directly.

But both lettre and our fork don't wrap attachments:
- https://github.com/deltachat/lettre/blob/b8bef0b6c168557aa1bdc566bc27b23b89f2a229/lettre_email/src/lib.rs#L273
- https://github.com/lettre/lettre/blob/3995ea298359031bcba3569e2af56195003c9095/src/builder/mod.rs#L316